src/RouteAttributesRegistrar.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Router;
+
+/**
+ * Provides an implementation of {@see RouteAttributesRegistrarInterface} using {@see get_declared_classes()} function.
+ */
+final class RouteAttributesRegistrar implements RouteAttributesRegistrarInterface
+{
+    public function __construct(private RouteCollectorInterface $routeCollector)
+    {
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function register(): void
+    {
+        //TODO: caching?
```

Comment:
?

Reply:
Reflection caching

Reply:
Either we need to remove that TODO and make it an issue in this repo or we need to implement that in this pull request.